apps/zhp-accounts-backend/Taskfile.yml

```diff
@@ -0,0 +1,23 @@
+version: '3'
+
+includes:
+
+tasks:
+    default:
+        desc: Help
+        summary: Show available tasks
+        aliases:
+            - help
+        cmd: task --list-all
+
+    install:
+        desc: Install dependencies
+        summary: Install Python dependencies
+        cmd: pip install -r requirements.txt
+
+    start:local:
+        desc: Run locally
+        summary: Run the functions locally (requires azure-functions-core-tools)
+        cmd: func start
+        preconditions:
+            - func --version
```

Comment:
on nie powinien też może obsłużyć venv?

Reply:
W sensie? Żeby uruchomić venv przed uruchomieniem funkcji?

Reply:
Hmm, próbuję na różne sposoby do tego dojść i mam z tym problem, przez to, że pomiędzy wykonywanymi komendami wygląda, jakbym nie mógł utrzymać "stanu" shella.
Jak uruchamiam sobie np. `powershell.exe ".\\.venv\\Scripts\\activate; echo '$VIRTUAL_ENV'"` w konsoli (bo to jest narazie "najbliższe" zadziałaniu), to mi wyświetla na spokojnie lokalizację VIRTUAL_ENV. Jak uruchamiam taką samą komendę przez `task start:local`, to wyświetla pustą linię, jakby VIRTUAL_ENV się nie przechowywało pomiędzy uruchomieniami. Trochę nie wiem, jak do tego dojść, żeby to działało w ten sposób

Reply:
na ten moment zrobię merge - to nie jest raczej nic super blokującego; będzie można to zmodyfikować w późniejszym czasie, a na ten moment będziemy mogli pójść z budowaniem backendu